Woodoku/src/Entity/WoodokuBoard.py

```diff
@@ -13,16 +16,37 @@ class _WoodokuBoardRepresentation:
     The last block on the top row is (0, n - 1).
     The first block on the bottom row is (n - 1, 0).
     The bottom-right block's coordinate is (n - 1, n - 1).
+
+    _board: an 2d array to record the occupancy of each position on the game board. The value is set to True when
+    the position occupied
     """
+    __board: ndarray
 
     def __init__(self):
+        self.__board = np.full((N, N), False)
+
+    def __str__(self) -> str:
         pass
```

Comment:
Style comment: all __someting__ function except __init__ should the last methods of the class